src/data/ua_overrides.js

```diff
@@ -607,6 +607,25 @@ const AVAILABLE_UA_OVERRIDES = [
       },
     },
   },
+  {
+    /*
+     * Bug 1622063 - UA overrides for bracketchallenge.ncaa.com
+     * Webcompat issue #49886 - https://webcompat.com/issues/49886
```

Comment:
should be wp1-ext.usps.gov? and webcompat issue #29867

Reply:
argh, rookie move. thanks :)